assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -221,33 +224,29 @@ function MediaPane(props) {
     <StyledPane id={paneId} {...props}>
       <Inner>
         <Padding>
-          <Header>
-            <Title>
-              {__('Media', 'web-stories')}
-              {(!isMediaLoaded || isMediaLoading) && <Spinner />}
-            </Title>
-            <MainButton onClick={openMediaPicker}>
-              {__('Upload', 'web-stories')}
-            </MainButton>
-          </Header>
+          {(!isMediaLoaded || isMediaLoading) && <Spinner />}
 
           <SearchInput
             value={searchTerm}
-            placeholder={__('Search media...', 'web-stories')}
+            placeholder={__('Search', 'web-stories')}
             onChange={onSearch}
           />
-
-          <FilterButtons>
-            {FILTERS.map(({ filter, name }, index) => (
-              <FilterButton
-                key={index}
-                active={filter === mediaType}
-                onClick={onFilter(filter)}
-              >
-                {name}
-              </FilterButton>
-            ))}
-          </FilterButtons>
+          <FilterArea>
+            <FilterButtons>
+              {FILTERS.map(({ filter, name }, index) => (
+                <FilterButton
+                  key={index}
```

Comment:
I’m guessing there isn’t anything more unique than the array index to use as the key?

Reply:
good nit, but I only reshuffled this code, didn't add it.. not sure who did originally, maybe @spacedmonkey knows

Reply:
Not my code but: These elements never change their position, so it's perfectly safe to use `index`, CMIIW.